nipype/algorithms/tests/test_compcor.py

```diff
@@ -0,0 +1,92 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import nipype
+from ...testing import assert_equal, assert_true, assert_false, skipif, utils
+from ..compcor import CompCor, TCompCor, ACompCor
+
+import unittest
+import mock
+import nibabel as nb
+import numpy as np
+import os
+
+class TestCompCor(unittest.TestCase):
+    ''' Note: Tests currently do a poor job of testing functionality '''
+
+    filenames = {'functionalnii': 'compcorfunc.nii',
+                 'masknii': 'compcormask.nii',
+                 'components_file': None}
```

Comment:
It would be better to create the test file int temp storage (see https://docs.python.org/3/library/tempfile.html) rather than current workign directory. Even though they are removed after tests run, if the testing suite crashes or gets interrupted the code directories will get polluted with temp files.